detect_alpine_image

```diff
@@ -6,7 +6,7 @@ img="unknown"
 if [ "$arch" = "x86_64" ]; then
     img="alpine"
 elif [ "$arch" = "armv7l" ]; then
-    img="armhf/alpine"
+    img="arm32v7/alpine"
```

Comment:
So this one needs to be `v6`, but the rest should be `v7` as you have them. :+1:

Reply:
Oh, and the `aarch64/` below should be `arm64v8/` instead, but perhaps that's a separate PR? :smile:

Reply:
Ah, yes I can change arm64 as well

Thanks for reviewing, I'll update the alpine one